how-to/integrate-with-sso/integrate-with-auth0/package.json

```diff
@@ -1,20 +1,20 @@
 {
-	"name": "openfin-workspace--integrate-with-sso",
+	"name": "openfin-workspace--integrate-with-auth0",
 	"version": "10.3.0",
```

Comment:
This has been updated in main so I don't know if a resync with main will pick up the workspace 11 changes. If not then we just need to update the version number to 11.0.0 and update the dependencies to reflect workspace 11.